soccer/src/soccer/strategy/coach/coach_node.hpp

```diff
@@ -51,6 +60,12 @@ class CoachNode : public rclcpp::Node {
     void world_state_callback(const rj_msgs::msg::WorldState::SharedPtr msg);
     void ball_sense_callback(const rj_msgs::msg::RobotStatus::SharedPtr msg, bool our_team);
     void check_for_play_state_change();
+
+    /*
+     * Assigns positions to robots with IDs 1 through 16, depending on current possession.
+     * Publishes new positions to the topic /strategy/positions (message type Position)
+     */
```

Comment:
maybe note that this is temporary, as in the future we would like the coach to assign positions based on the play state

Reply:
EDIT: I misunderstood this comment, got it